components/scrollable-container/demo/demo.html

```diff
@@ -3,12 +3,12 @@
 <html lang="en">
 <head>
     <link rel="stylesheet" href="../components-theme.css">
```

Comment:
We forgot to discuss this, but should we also change the theme name to coherent-gameface-components-theme.css?